plugins/kotlin/uast/uast-kotlin-fir/src/org/jetbrains/uast/kotlin/FirKotlinUastResolveProviderService.kt

```diff
@@ -77,18 +77,19 @@ interface FirKotlinUastResolveProviderService : BaseKotlinUastResolveProviderSer
     override fun findDefaultValueForAnnotationAttribute(ktCallElement: KtCallElement, name: String): KtExpression? {
         analyseForUast(ktCallElement) {
             val resolvedAnnotationConstructorSymbol =
-                ktCallElement.resolveCall()?.targetFunction?.getSingleCandidateSymbolOrNull() as? KtConstructorSymbol ?: return null
+                ktCallElement.resolveCall().singleCallOrNull<KtFunctionCall<*>>()?.symbol as? KtConstructorSymbol ?: return null
```

Comment:
That would overlap a bit with `KtAnnotationCall` since both would call constructors. I kind of prefer keeping `KtCall` data model minimal and delegate more information about the symbol kind to the actual symbol. The reason I am keeping `KtAnnotaitonCall` and `KtDelegatedConstructorCall` is because they are somewhat special (different syntax/requirement at call site) where constructors are just called like functions.